### PR TITLE
Updated tensorflow version

### PR DESCRIPTION
**Pull request by happyhuman**

### CODE REVIEW

**Comment by happyhuman** [comment]
I tested the census code locally with tensorflow==1.8 and I had no issues. We should be able to merge this branch @puneith

**Comment by puneith** [comment]
Does it run on CMLE? While you are at it we might benefit from writing few e2e tests in this script https://github.com/GoogleCloudPlatform/cloudml-samples/blob/master/census/sample.sh

So long as you write the commands (which you anyways will run to test this change) in this file it will do the trick. This will be run by the Kokoro testing system and any failures will be reported in the PR. Additionally I think there is a bug right now and failures are seen in the logs but not reported due to wrong bash error code being returned by the script.

**Comment by happyhuman** [comment]
I tested it locally and using gcloud and it worked fine. I did not test it beyond these two cases.
Sure, I can take a look at the e2e tests you mentioned.

**Comment by happyhuman** [comment]
Come to think of it, I think we want to add the e2e tests in a separate PR and not in this one. Would it make sense to close this PR and create a whole different one for adding the tests @puneith ?

**Comment by puneith** [comment]
My reasoning for this one was because this is a very minimalistic PR and the commands you used to verify it on CMLE is what we can simply paste in the .sh script and get done with it. If you think you can get to that in a different PR then sure :)

**Comment by happyhuman** [comment]
@puneith I believe sample.sh is good now. It runs both estimator and customestimator and it returns the exit values properly.

File: census/sample.sh [hunk 1]
```diff
@@ -13,31 +13,43 @@
...
+	echo "python script failed"
+	return 1
+}
+
+run_script_local estimator
+if [ $? = 1 ]; then
+	exit 1
+fi
+
+run_script_local customestimator
```
[the remark this answers is not in the document]
Comment (happyhuman) [reply census/sample.sh]: You mean the command from the "Using Cloud ML Engine" section in the README file?

[the remark this answers is not in the document]
Comment (happyhuman) [reply census/sample.sh]: Just added it, @puneith

**Comment by puneith** [comment]
Thanks Shahin, this is a very impactful change. 👍 LGTM!

**Pull request merged**